databricks/koalas/tests/test_dataframe.py

```diff
@@ -345,6 +345,51 @@ def test_abs(self):
         self.assert_eq(ddf[['B', 'C']].abs(), df[['B', 'C']].abs())
         # self.assert_eq(ddf.select('A', 'B').abs(), df[['A', 'B']].abs())
 
+    def test_corr(self):
```

Comment:
i'd create a new file for these functions. in the future we can parallelize these tests and if they are different files it's easier to do that.